File: src/main/java/com/examples/printsum/correct/java/withoronnewline/Question.java
package com.examples.printsum.correct.java.withoronnewline;

import edu.illinois.cs.cs125.questioner.lib.AlsoCorrect;

@SuppressWarnings("ConstantConditions")
@AlsoCorrect
public class Question {
  public static void printSum(int first, int second) {
    // TEMPLATE_START
    if (first != second || first == second) {
      System.out.println(first + second);
    }
    // TEMPLATE_END
  }
}

File: src/main/java/com/examples/inputinterleavingtest/correct/kotlin/Question.kt
@file:AlsoCorrect

package com.examples.inputinterleavingtest.correct.kotlin

import edu.illinois.cs.cs125.questioner.lib.AlsoCorrect
import java.util.Scanner

/*
 * Testing STDIN generation and interleaving. In Kotlin!
 */

fun echo() {
  println("Hello!")
  val scanner = Scanner(System.`in`)
  val first = scanner.nextLine()
  println("First: $first")
  val second = scanner.nextLine()
  println("Second: $second")
}